hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/http/DefaultSendingResultHandlers.java

```diff
@@ -19,7 +19,7 @@ public Response.CompleteListener handleSendingResultForBroadcast(CompletableFutu
     }
 
     @Override
-    public MessageSendingResult handleSendingResultForBatch(CloseableHttpResponse response) {
-        return MessageSendingResult.ofStatusCode(response.getStatusLine().getStatusCode());
+    public MessageSendingResult handleSendingResultForBatch(ContentResponse response) {
```

Comment:
I wonder if it is feasible to make this API simpler - ideally, I would like to have one handler for all types of subscriptions. 

This is not strictly related to this PR. I'm just curious if we can somehow leverage the fact that now we use the same HTTP client everywhere. Plus, I'm not sure if we have the capacity for a bigger refactoring. Let's discuss this next week.